Sources/Parsers/AssetsCatalogParser.swift

```diff
@@ -104,6 +107,9 @@ extension AssetsCatalogParser {
     let type = item.extension ?? ""
 
     switch (type, AssetCatalog.Item.supported.contains(type)) {
```

Comment:
I've implemented option 2 for now, but I think we're better off long term using option 1?